ducktape/command_line/main.py

```diff
@@ -24,10 +23,15 @@
 import argparse
 import os
 import sys
-
+import importlib
+import itertools
 
 def parse_args():
-    """Parse in command-line options.
+    """Parse in command-line and config file options.
+
+    Command line arguments have the highest priority, then user configs specified in ~/.ducktape/config, and finally
```

Comment:
Hm, is adding extra state outside of the project that useful? I guess if you're using it with several different projects it's nice to have ducktape work the way you want out-of-the-box is nice.